### DISCUSSION TITLE
Send Vercel logs to Coralogix

### DISCUSSION THREAD

**Discussion by yahel2410** [vercel/community#4627] [Feature Question]
### Question

Hi, 

We are using [Coralogix](https://coralogix.com/) as our logging platform.

Is it possible to send Vercel logs to Coralogix?

Having trouble setting up a Log Drain since Coralogix doesn't support POST of arbitrary JSON logs.

**Comment by amyegan** [comment]
We have an [OpenTelemetry Collector](https://vercel.com/docs/observability/otel-overview/quickstart#) and [Log Drains](https://vercel.com/docs/observability/log-drains-overview/log-drains#) to help you collect all of your logs and OTEL traces. If those won't work for you, and you're open to using a different logging platform, there are also some [logging integrations](https://vercel.com/integrations#logging) available.

**Comment by yahel2410** [comment]
Thank you for your response, @amyegan.

AFAIK, the OTEL collection is used for metrics rather than logs. Correct me if I'm wrong. 
Log Drains did not work for me, as I mentioned earlier. 
Replacing our logging plaform is not feasiable since we have a lot of other apps beside Vercel that are already configured.

**Comment by denzhel** [comment]
Hi, this is very urgent for us.
Any news regarding this issue ?

**Comment by EvyatarVanunu1** [comment]
Hi, 
We are also facing the same issue. Any updates?

**Comment by golankopi** [comment]
hi any update on this issue?
